doc/classes/EditorSettings.xml

```diff
@@ -1063,10 +1063,16 @@
 		<member name="text_editor/behavior/navigation/stay_in_script_editor_on_node_selected" type="bool" setter="" getter="">
 			If [code]true[/code], prevents automatically switching between the Script and 2D/3D screens when selecting a node in the Scene tree dock.
 		</member>
+		<member name="text_editor/behavior/navigation/use_custom_separators" type="bool" setter="" getter="">
+			If [code]false[/code], using [kbd]Ctrl + Left[/kbd] or [kbd]Ctrl + Right[/kbd] ([kbd]Cmd + Left[/kbd] or [kbd]Cmd + Right[/kbd] on macOS) bindings will stop moving caret only if a space or punctuation is detected. If [code]true[/code], it will also stop the caret if a character within [member text_editor/behavior/navigation/word_separators] is detected. Useful for subword moving. This behaviour also will be applied to the behaviour of text selection.
```

Comment:
```suggestion
			If [code]false[/code], using [kbd]Ctrl + Left[/kbd] or [kbd]Ctrl + Right[/kbd] ([kbd]Cmd + Left[/kbd] or [kbd]Cmd + Right[/kbd] on macOS) bindings will stop moving caret only if a space or punctuation is detected. If [code]true[/code], it will also stop the caret if a character within [member text_editor/behavior/navigation/word_separators] is detected. Useful for subword moving. This behavior also will be applied to the behavior of text selection.
```
Same in `TextEdit`. We use American English spelling for the docs.